apps/web/src/app/(dashboard)/admin/stats/user-with-document-cummulative.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { DateTime } from 'luxon';
+import { Bar, BarChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
+
+import type { GetUserWithDocumentMonthlyGrowth } from '@documenso/lib/server-only/admin/get-users-stats';
+
+export type UserWithDocumentCummulativeChartProps = {
+  className?: string;
+  data: GetUserWithDocumentMonthlyGrowth;
+};
+
+export const UserWithDocumentCummulativeChart = ({
+  className,
+  data,
+}: UserWithDocumentCummulativeChartProps) => {
+  const formattedData = [...data]
+    .reverse()
```

Comment:
I assume you're spreading to not mutate the array

You can do data.map().reverse() which should be the same thing

Reply:
sure, I'll update that

Reply:
doing that, I realised that without spreading, the reverse does not work, reverse works only when the data is being spread into a new array. i'm going to maintain the old one